about.html

```diff
@@ -0,0 +1,234 @@
+                </div>
+        
+            </div>
+
+            <div class="row pb-5 align-items-center text-center justify-content-center">
+                <div class=" col-md-4 summit-logo p-2">
+                   <img id="summit" src="assets/images/logo/Tech.png" alt="techlife logo" class="img-fluid">
+                </div> 
+            </div>
+        </div>
+        <hr>
+    </section>
+
+    <!-- feature speakers -->
+    <section id="past-summit">
+        <div class="container-lg">
+            <div class="text-center">
+                <h2 class="summit-heading pt-4 pb-2">See the Past TechLife Global Summits</h2>
+                <div class="line"></div>
+                <p class="past-p p-3">Take a look at the past two TechLife Global Summits which took place in Herat and in Kabul </p>
+            </div>
+            <div class="row align-items-center justify-content-center pb-5 pt-2 g-1">
+
+                <div class="col-md-4 me-2 text-center">
+                    <div class="img-fluid">
+                        <div class="img-container">
+                            <img src="assets/background/past-summit-1.png" alt="Techlife 2021 summit" class="img-fluid">
+
+                            <div class="content pt-5">
+                              <h2 class="img-header">2021</h2>
+                              <p class="img-text">Techlife Innovative Global Summit in 2021</p>
+                            </div>
+                        </div>
+                    </div>              
+                </div>  
+
+                <div class="col-md-4 me-2 text-center">
+                    <div class="img-fluid">
+                        <div class="img-container">
+                            <img src="assets/background/past-summit-2.png" alt="Techlife 2021 summit" class="img-fluid">
+
+                            <div class="content pt-5">
+                              <h2 class="img-header">2022</h2>
+                              <p class="img-text">Techlife Innovative Global Summit in 2022</p>
+                            </div>
+                        </div>
+                    </div>              
+                </div>   
+            </div>
+        </div>
+    </section>
+
+    <!-- footer  -->
+
+    <footer id="footer" class="hideInMobile">
+        <div class="container-fluid about-footer py-5">
+                <div class="col-md-8 offset-md-2">
+                    <div class="footer-p-logo">
+                        <div class="footer-logo">
+                            <img src="assets/images/logo/techlife.png" alt="mozilla" class="img-fluid logo">
+                        </div>
+                        <p class="about-footer-p footer-p">2023 Creative Commons Korea. Some Rights Reserved.</p>
+                    </div>
+                </div> 
+        </div>
+    </footer>
+
+    <!-- show in mobile -->
+
+    
+    <!-- partners  -->
+    <section id="partners" class="showInMobile">
```

Comment:
- [ ] Good job so far but I think you can improve this section's design by reducing the partner's lo image and aligning them as you can see on the original design. You can check the screenshots below 👇  to see the differences. As mentioned by the previous reviewer it is very important to stick to the original design as much as possible. Please fix it 👍 

#### Your design
![partner](https://github.com/MozamelJawad/Capstone-Project/assets/6615394/88b9d2ad-8dc0-4b98-96cc-ff6ef5618872)


#### Original design
![partner-o](https://github.com/MozamelJawad/Capstone-Project/assets/6615394/2afe2f6c-8a39-48e9-b095-67e730f1f479)